### DISCUSSION TITLE
Gitlab and `footer`

### DISCUSSION THREAD

**Discussion by grzesuav** [goreleaser/goreleaser#4680] [Q&A] [answered]
hi, I tried to use `footer` on gitlab, however it does not work. I do not see it also in documentation, is it not implemented for gitlab ?

**Comment by caarlos0** [comment]
it should work, yes

**Reply by grzesuav** [reply]
So, it does not work, example release https://gitlab.com/metacontroller/test-sem-rel/-/releases,

Configuration https://gitlab.com/metacontroller/test-sem-rel/-/blob/main/.goreleaser.yaml?ref_type=heads#L139

**Reply by caarlos0** [reply]
I just tested on one of my repos, it does work.


maybe run with `--verbose` (in a private repo, it might leak info) and see what's going on?

per what I investigated, it works fine. Even tried creating the release manually first and then running goreleaser, and still everything worked as expected...

**Reply by caarlos0** [reply]
I tested on both github and gitlab, fwiw

**Reply by grzesuav** [reply]
Huh, will try, will let you know about the result

**Reply by grzesuav** [reply]
btw any chance footer example can be added to https://gitlab.com/goreleaser/example/ ?

**Reply by caarlos0** [reply]
sure... do you want to PR it?

**Reply by grzesuav** [reply, marked as answer]
can do !

Btw I just checked, and it seems some `go-semantic-release` hook implementation, it seems whole `goreleaser` is embedded there. When I separated to phases:
* `go-semantic-release` runs on `main` and just create tags
* `goreleaser` runs on tags and do the stuff
it started to work - https://gitlab.com/metacontroller/test-sem-rel/-/releases/v1.3.6 - however I switch to use `$CI_JOB_TOKEN` at the same moment, which failed in first run, when I protected branches it started to work

**Reply by grzesuav** [reply]
https://gitlab.com/goreleaser/example/-/merge_requests/2

**Reply by caarlos0** [reply]
I wonder what go-semantic-release is doing to make it not work? 

does go-semantic-release updates the release body again after goreleaser run?

**Reply by grzesuav** [reply]
From what I can see https://github.com/go-semantic-release/hooks-goreleaser/blob/main/pkg/hooks/goreleaser.go it wraps it execution into own hook and run grammatically setting some parameters, so it would be difficult for me to say where exactly issue come from

**Reply by caarlos0** [reply]
oh, yeah, it seems to have goreleaser's code embedded there, and def its changing the configuration there as well...

thanks for bearing with me investigating this, btw!
